src/Sulu/Bundle/MediaBundle/Tests/Functional/Controller/CollectionControllerTest.php

```diff
@@ -586,10 +629,14 @@ public function testPut()
 
         $this->assertNotEmpty($response);
 
-        $this->assertEquals(1, $response->total);
+        $this->assertEquals(2, $response->total); // 1 created 1 system collection
```

Comment:
Isn't it possible to change the configuration in a way that it doesn't create any system collections for this test?